{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -96,8 +102,8 @@ tests_require=
     flake8
     pytest-flask>=0.10.0
     sphinxcontrib-napoleon
-    gunicorn>=0.19
```

Comment:
all the logs formating in json and configuration is done with gunicorn for now. Until replaced with an alternative should keep it